src/test-runs/dto/create-test-request-multipart.dto.ts

```diff
@@ -25,7 +27,7 @@ export class CreateTestRequestMultipartDto extends OmitType(CreateTestRequestDto
   @Transform(({ value }) => value == 'true')
   merge?: boolean;
 
-  @ApiPropertyOptional({ type: String })
+  @ApiPropertyOptional({ type: String, isArray: true })
```

Comment:
the input should remain `String` only transform makes it as array 
seems like linter is not able to catch such cases